setup.py

```diff
@@ -0,0 +1,30 @@
+import pathlib
+from setuptools import setup, find_packages
+
+# The directory containing this file
+HERE = pathlib.Path(__file__).parent
+
+# The text of the README file
+README = (HERE / "README.md").read_text()
+
+# This call to setup() does all the work
+setup(
+    name="thermal-base",
+    version="1.0.0",
+    description="Tools for operating on thermal images",

```

Comment:
*Decoding and Basic processing of thermal images (The tools would go into the other repo)